### DISCUSSION TITLE
Can't open documents with collabora

### DISCUSSION THREAD

**Discussion by tijnsc** [nextcloud/all-in-one#3995] [Questions] [answered]
Hello,
After restoring a backup during a storage migration I am no longer able to open documents with collabora.
Portainer marks the apache container as unhealthy, but no errors show in that log.
In the nextcloud log I constantly get this error:
```
GuzzleHttp\Exception\ConnectException: cURL  error 7: Failed to connect to domain.duckdns.org port 443 after 2  ms: Couldn't connect to server (see  https://curl.haxx.se/libcurl/c/libcurl-errors.html) for  https://domain.duckdns.org/hosting/capabilities
```
and in my nextcloud container log I get:
```
Activating Collabora config...
Failed to activate any config changes
cURL error 7: Failed to connect to domain.duckdns.org port 443 after 1 ms: Couldn't connect to server (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://domain.duckdns.org/hosting/discovery
```

This problem stops when I disable the collabora container from the AIO interface.

I have tried disabling ufw, reverting to a backup when it was still working and upgrading to the beta release channel. I also tried fully resetting the instance and reverting to a backup.

I am running nextcloud-aio as a Portainer stack from a modified version of the compose.yaml provided in the all-in-one repository.
The server is running behind nginx proxy manager and with a duckdns domain.

I hope someone is able to help me solve this issue.

**Comment by elephantastyczny** [comment]
Are you sure that your NC domain is domain.duckdns.org?
You should check your compose.yaml file.

**Comment by szaimen** [comment, marked as answer]
Hi @tijnsc can you follow https://github.com/nextcloud/all-in-one/discussions/1358?

**Reply by tijnsc** [reply]
Thank you for your reply, removing the hostname from /etc/hosts worked.
